### ISSUE TITLE
Xamarin Forms - 'System.UnauthorizedAccessException' in System.Private.CoreLib.dll

### ISSUE THREAD

**Issue by vsfeedback** [xamarin/Xamarin.Forms#12420]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/content/problem/1207940/xamarin-forms-systemunauthorizedaccessexception-in.html)._

---
[severity:Other]
Having just update Xamarin Forms to v4.8.0.1451, my project is encountering the following error, "Invalid URI: The format of the URI could not be determined." when debugging the UWP project. The functionality was OK before the upgrade. Looking at output the exception is:

```
Exception thrown: 'System.UriFormatException' in System.Private.Uri.dll
'Sentry.UWP.exe' (CoreCLR: CoreCLR_UWP_Domain): Loaded 'C:\Program Files\WindowsApps\Microsoft.NET.CoreFramework.Debug.2.2_2.2.28529.0_x86__8wekyb3d8bbwe\System.Diagnostics.StackTrace.dll'. Skipped loading symbols. Module is optimized and the debugger option 'Just My Code' is enabled.
'Sentry.UWP.exe' (CoreCLR: CoreCLR_UWP_Domain): Loaded 'C:\Program Files\WindowsApps\Microsoft.NET.CoreFramework.Debug.2.2_2.2.28529.0_x86__8wekyb3d8bbwe\System.IO.FileSystem.dll'. Skipped loading symbols. Module is optimized and the debugger option 'Just My Code' is enabled.
<p>System.UriFormatException: Invalid URI: The format of the URI could not be determined.
```
The output is attached. The functionality is OK with the upgrade,

---
#### Feedback Bot on 10/3/2020, 01:00 AM: 

<p>We have directed your feedback to the appropriate engineering team for further evaluation. The team will review the feedback and notify you about the next steps.</p>

---
### Original Solutions
(no solutions)

**Comment by MagicAndre1981** [comment]
Duplicate of #12153 and fixed in [4.8.0.1534 (4.8 SR3)](https://www.nuget.org/packages/Xamarin.Forms/4.8.0.1534)

**Comment by jsuarezruiz** [comment]
Seems to be the same as https://github.com/xamarin/Xamarin.Forms/issues/12153.
As @MagicAndre1981 mentioned, fixed in 4.8.0.1534
